app/models/reviewable.rb

```diff
@@ -101,7 +101,8 @@ def self.needs_review!(
     created_by:,
     payload: nil,
     reviewable_by_moderator: false,
-    potential_spam: true
+    potential_spam: true,
+    target_created_by: nil
```

Comment:
We need to be able to directly set `target_created_by` for `ReviewableQueuedPost`s  because,  the target (`Post`) isn't available during creation of the reviewable.